containers/pytorch/training/README.md

```diff
@@ -79,7 +79,7 @@ The PyTorch Training containers come with two different containers depending on
   docker build -t us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-pytorch-training-cu121.2-3.transformers.4-42.ubuntu2204.py310 -f containers/pytorch/training/gpu/2.3.0/transformers/4.42.3/py310/Dockerfile .
   ```
 
-- **TPU**: To build the PyTorch Training container for Google Cloud TPUs, an instance with at least one TPU available is required to install `optimum-tpu` which is a Python library with Google TPU optimizations for `transformers` models, making its integration seamless.
+- **TPU**: To build the PyTorch Training container for Google Cloud TPUs, no TPU is required. You can build it using this command:
```

Comment:
This sentence is strange, because it states that you do not need TPUs. Maybe you can rephrase it into something like  "You can build PyTorch Training container for Google Cloud TPUs on any machine with docker build, you do not need to build it on a TPU VM".

Reply:
Done, I used your sentence in the end